pkg/apis/acid.zalan.do/v1/crds.go

```diff
@@ -247,6 +247,91 @@ var PostgresCRDResourceValidation = apiextv1.CustomResourceValidation{
 							"user": {
 								Type: "string",
 							},
+							"nodeAffinity": {
+								Type: "object",
+								Properties: map[string]apiextv1.JSONSchemaProps{
+									"preferredDuringSchedulingIgnoredDuringExecution": {
+										Type: "array",
+										Items: &apiextv1.JSONSchemaPropsOrArray{
+											Schema: &apiextv1.JSONSchemaProps{
+												Type: "object",
+												Properties: map[string]apiextv1.JSONSchemaProps{
+													"preference": {
+														Type: "object",
+														Properties: map[string]apiextv1.JSONSchemaProps{
+															"matchExpressions": {
+																Type: "array",
+																Items: &apiextv1.JSONSchemaPropsOrArray{
+																	Schema: &apiextv1.JSONSchemaProps{
+																		Type: "object",
+																		AdditionalProperties: &apiextv1.JSONSchemaPropsOrBool{
+																			Allows: true,
+																		},
+																	},
+																},
+															},
+															"matchFields": {
+																Type: "array",
+																Items: &apiextv1.JSONSchemaPropsOrArray{
+																	Schema: &apiextv1.JSONSchemaProps{
+																		Type: "object",
+																		AdditionalProperties: &apiextv1.JSONSchemaPropsOrBool{
+																			Allows: true,
+																		},
+																	},
+																},
+															},
+														},
+													},
+													"weight": {
+														Type:   "integer",
+														Format: "int32",
+													},
+												},
+												Required: []string{"preference, weight"},
```

Comment:
can you also move it above `Properties`? I know, it's not alphabetical order, but improves readability a little.

Reply:
will do!